scout/server/blueprints/cases/templates/cases/collapsible_actionbar.html

```diff
@@ -276,16 +276,33 @@
 </a>
 <!-- Submenu content -->
 {% if case.rerun_requested %}
+  {% if config.SHOW_MELTER %}
```

Comment:
SHOW_MELTER is a bit tricky as a name for a config param. It's not self-explanatory and it has a meaning just for CG Solna. Rename it to something like `RERUN_MONITORING` ?